x/gov/client/cli/query.go

```diff
@@ -638,7 +641,7 @@ $ %s query gov param deposit
 				return fmt.Errorf("argument must be one of (voting|tallying|deposit), was %s", args[0])
 			}
 
-			return clientCtx.PrintOutput(out)
+			return clientCtx.PrintString(fmt.Sprintf("%s\n", out))
```

Comment:
PrintOutput instead?

Reply:
Sorry maybe this should just be `PrintOutputLegacy`